odl/solvers/functional/default_functionals.py

```diff
@@ -2239,10 +2307,10 @@ class MoreauEnvelope(Functional):
     References
     ----------
     .. _Proximal Algorithms: \
-https://web.stanford.edu/~boyd/papers/pdf/prox_algs.pdf
+    https://web.stanford.edu/~boyd/papers/pdf/prox_algs.pdf
     """
 
-    def __init__(self, functional, sigma=1.0):
+    def __init__(self, functional, sigma=1.0, range=RealNumbers()):
```

Comment:
Should be None

Reply:
When did we make that decision? I vaguely remember a discussion.